package exceptions;

public class ExceptionDemo {
	
	int a = 4;
	int b = 7;
	int c = 0;
	int k=b/c;
	try {
		
		
		System.out.println(k);
		
	} catch (Exception e) {
		
		System.out.println(e);
	
	}
	
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		ExceptionDemo e = new ExceptionDemo();
		System.out.println(e.k);
	}

}
